### PR TITLE
CONFIG env variable

### PR DESCRIPTION
**Pull request by 40818419**
Possibility to provide `CONFIG` over  node env.

### What is accomplished by your PR?

Default configuration is stored in `/src/app/config/default.json`, which can get overridden by providing valid JSON in the environment variable `CONFIG` on application startup or by an environment specific file.
For instance, in order to set a custom API endpoint, run `CONFIG='{"api":{"baseUrl": "http://custom.api.endpoint/"}}' npm run start`. Check documentation of [config](https://www.npmjs.com/package/config) module.

Also added to `README.md`

### Link to the Issue

#60

# Checklist

### New Feature / Bug Fix

- [x] Run unit tests to ensure all existing tests are still passing
- [ ] Add new passing unit tests to cover the code introduced by your PR

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -175,7 +176,7 @@
     "rimraf": "2.6.2",
     "sass-loader": "6.0.7",
     "serviceworker-webpack-plugin": "0.2.3",
-    "storybook-addon-vue-info": "0.1.8",
+    "storybook-addon-vue-info": "^0.2.0",
```
Comment (devCrossNet) [inline package.json]: can you remove the ^

Comment (40818419) [reply package.json]: i culdn't start the app with older version..

Comment (devCrossNet) [reply package.json]: the version is fine. it's just to pin the exact version in package.json

Comment (40818419) [reply package.json]: done!

**Pull request merged**